dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/Main.java

```diff
@@ -53,14 +57,36 @@
     private ResourceBundle resourceBundle = null;
     private Stage primaryStage = null;
     private JobInterruptionStore jobInterruptionStore = null;
-    private final EventHandler<WindowEvent> confirmCloseEventHandler = event -> {
 
-        if (jobWorkers.getTasks().isEmpty()) {
-            closeApplication();
-            event.consume();
 
-        } else {
+    /*************************/
+    private static final String WINDOW_POSITION_X = "Window_Position_X";
+    private static final String WINDOW_POSITION_Y = "Window_Position_Y";
+    private static final String WINDOW_WIDTH = "Window_Width";
+    private static final String WINDOW_HEIGHT = "Window_Height";
+    private static final double DEFAULT_X = 10;
+    private static final double DEFAULT_Y = 10;
+    private static final double DEFAULT_WIDTH = 800;
+    private static final double DEFAULT_HEIGHT = 600;
+    private static final String NODE_NAME = "ViewSwitcher";
+    private static final String BUNDLE = "Bundle";
+
+    /*************************/
+
+    Preferences pref = Preferences.userRoot().node(NODE_NAME);
```

Comment:
The code snippet has been moved from Main.java class to ApplicationPreferences.java, and the value has been defined as private static final.